configs/10.0/packages/expat/stage_1

```diff
@@ -22,12 +22,30 @@
 ATCFG_HOLD_TEMP_INSTALL='no'
 ATCFG_HOLD_TEMP_BUILD='no'
 # Build in a new directory.
-ATCFG_BUILD_STAGE_T='dir'
+ATCFG_BUILD_STAGE_T='link'
```

Comment:
@er-1 Is this really necessary?  I thought expat supported VPATH.

Reply:
@tuliom Good point. The change comes from what we have had in AT 11.0.